ores/score_caches/tests/test_redis.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from ..redis import Redis
+
+
+@pytest.fixture
+def cache():
+    return Redis.from_config({
+        'score_caches': {
+            'redis': {
+                'host': 'localhost',
+                'prefix': 'ores-derp',
+                'ttl': 100}}}, 'redis')
```

Comment:
Assumes that redis is installed on the test machine.  Maybe we should catch the error we expect if redis is not available and raise a meaningful error message saying "Redis must be installed and available via 'localhost'".

Reply:
The problem is that these fixtures only provide the class and don't do anything to check it against and see if redis is not setup. If you stop redis, tests fail with "redis.exceptions.ConnectionError: Error 111 connecting to localhost:6379. Connection refused." which is not that bad. One way to make it better to wrap all of redis tests into a try but I'm not sure if it's worth the trouble. I add pytest.mark('redis') to it so we can easily skip all those though. That would help a lot.

Reply:
This is complicated.  We don't usually rely on external services in unit tests because it complicates things.  What exactly are we trying to test here?  Could we test it without using the redis service directly?  We can always mock a redis connection pretty easily and check that connecting to redis actually works when we stand a real instance of the service up.

Reply:
It will be super hard to implement such measures + all of these tests are marked as "redis" type so in case redis is not available you can just skip them by running "pytest -m "not redis"". We depend on enchant libraries on our tests for revscoring.

Reply:
I agree that we should only mock here, but also fine with the integration tests as written.